src/main/java/dev/sefiraat/cultivation/implementation/utils/GeneralUtils.java

```diff
@@ -0,0 +1,14 @@
+package dev.sefiraat.cultivation.implementation.utils;
+
+import org.bukkit.Bukkit;
+
+public class GeneralUtils {
+
+    private GeneralUtils() {
+        throw new IllegalStateException("Utility class");
+    }
+
+    public static int getMajorServerVersion() {
+        return Integer.parseInt(Bukkit.getServer().getBukkitVersion().split("\\.")[1]);
```

Comment:
since this is a Slimefun addon, just use the PaperLib's getMajorVersion or Slimefun.getMinecraftVersion()?

Reply:
oh! I wasn’t familiar with that method from paper lib, and I thought the mc version enum didn’t do minor versions, ill look into it when im back at my computer

Reply:
Yep looks like I can use PaperLib#getMinecraftVersion >= 20
I'll replace it with that when I'm at my comp